cpu/atmega_common/periph/rtt.c

```diff
@@ -95,13 +182,42 @@ static inline uint8_t _rtt_div(uint16_t freq)
     default   : assert(0);
     }
 }
+#endif
 
-void rtt_init(void)
+static inline void _timer_init(void)
```

Comment:
You can put the `_timer_init()` for TIMER2 in the same `#ifdef` block as `_rtt_div()`.